.stylelintignore

```diff
@@ -1 +0,0 @@
-/app/styles/basscss/
```

Comment:
This file was not used by `ember-cli` for some reasons. Direct run of `stylelint`, though, demonstrated the correct behavior. 
I decided to move this line into the config file where it works for both cases.